src/main/scala/chisel3/UnitTestMain.scala

```diff
@@ -12,6 +12,7 @@ object UnitTests {
 
   /** Command line configuration options. */
   case class Config(
+    runpath:    List[String] = List(),
```

Comment:
`case class`es are basically unusable from a binary compatibility perspective. Does this need to be public or can we make it private?

As this has not gotten usage, I'm fine to just make this private and merge it ignoring the checks. @jackkoenig WDYT?

Reply:
I was about to say basically this. Even though I can't see why anyone would use this, I don't want to break binary compatibility because they "probably haven't".

Making it private is still a hard break but at least it's a hard break rather than a subtle runtime linkage issue. So I think that's fine.

A little bit better would be to just copy-paste it, keep the old one around deprecated saying something like "this isn't used in public APIs" and create a new one that is private that you actually use here. A little bit more stuff but not _that_ much.